test/proposal.test.ts

```diff
@@ -216,7 +239,26 @@ describe("proposal", function () {
         const totalSupply = await contracts.cowToken.totalSupply();
         expect(
           await contracts.cowToken.balanceOf(contracts.cowDao.address),
-        ).to.equal(totalSupply);
+        ).to.equal(totalSupply.sub(settings.bridge.amountToRelay));
+      });
+
+      it("has approval to send tokens to bridge", async function () {
+        expect(
+          await contracts.cowToken.allowance(
+            gnosisDao.address,
+            multiTokenMediatorETH.address,
+          ),
+        ).to.equal(settings.bridge.amountToRelay);
+      });
+    });
+
+    describe("Attention: Following test is missing: bridge relay", function () {
```

Comment:
This is pretty bad. But I think we can test it best via mainnet forking. This allows the best confirmation that all our proposed tx do what they should

It's hard to unit test a function that returns nothing. Maybe you guys have a good idea on how to do it.